framework/doc/content/source/userobject/NearestPointLayeredIntegral.md

```diff
@@ -0,0 +1 @@
+!template load file=stubs/moose_object.md.template name=NearestPointLayeredIntegral syntax=/UserObjects/NearestPointLayeredIntegral
```

Comment:
Can you add a short description to the documentation?

Reply:
Just did. Please let me know if that looks good.